core/src/main/java/org/elasticsearch/search/aggregations/metrics/stats/StatsAggregator.java

```diff
@@ -43,7 +43,7 @@
 /**
  *
  */
-public class StatsAggegator extends NumericMetricsAggregator.MultiValue {
+public class StatsAggregator extends NumericMetricsAggregator.MultiValue {
```

Comment:
This change is not related to the remainder of this pull request; can you please remove it from this change set?